coalib/parsing/ConfParser.py

```diff
@@ -15,7 +15,9 @@ def __init__(self,
                  key_delimiters=(',', ' '),
                  section_name_surroundings=None,
                  remove_empty_iter_elements=True):
-        section_name_surroundings = section_name_surroundings or {"[": "]"}
+        section_name_surroundings = (
+                {"[": "]"} if section_name_surroundings is None
+                else section_name_surroundings)
```

Comment:
maybe we could use `MappingProxyType` in this case and inline that into the signature^^